proposal.rst

```diff
@@ -267,8 +292,69 @@ Project Timeline
 .. If you will have other commitments during the project time (for example,
 .. any university classes or exams, vacations, etc), make sure you include them
 .. in your project timeline.
-
-FILLME
+In this summer I would like to do three things:
+
+    1. Make the go bindings compatible with the xapian build system.
+
+    2.  Provide bindings in go and complete API reference (as it is provided for C++)
+        using godoc,Go lang's official documentation tool where any one can look at the 
+        documentation if they just have go installed.The user can view the xapian 
+        documentation by just running go doc xapian from the terminal or command prompt.
+        And the documentation would be providied in godoc official website 
+        and xapian.org.
+        I have divided this part of work into to 10 weeks.
+
+    3.  Provide the go get feature for using xapian bindings in go with 
+        proper guide lines regarding whether the xapian core is installed
+        or not.
+
+
+Before May 18,2020 :
+I would finish up the build system part of work for go bindings that 
+are generated purely from xapian-headers.i and xapian-head.i such that
+these can be used like any other bindings and in any platform that other xapian bindings can be used in.
```

Comment:
A word of warning: unless you have access to a Windows machine (or VM) it may be difficult to get this working. There's a good chance that the go build system will have wrinkles specific Windows. What I'd suggest is that you move that to a stretch goal to the overall project. (Fully-functional go bindings that work on linux alone is better than partially-functional bindings that work on linux, Windows & macos.)

Reply:
It would certainly be good to get them working portably, but I'd agree that getting them working well on one platform first is probably a better approach.  Windows and macOS both have a number of differences from Unix-like platforms that can suck up a lot of time.